cli/cmd/volume.go

```diff
@@ -152,16 +154,44 @@ var volumeResetCmd = &cobra.Command{
 	},
 }
 
+func volumeInfoHandler2(cmd *cobra.Command)(api.Volinfo, error) {
```

Comment:
add one more argument to decide info or list. Example "isInfo bool"

Reply:
good idea :) implemented the same.